docs/monthly-meeting/2023-05.md

```diff
@@ -0,0 +1,121 @@
+* would be nice to have an ORG account.
+* curious about workflow for translations, etc how to migrate the docs translations and the multi-version support to readthedocs
+  * Manuel set up [python-docs-es.readthedocs.io, the Spanish translation](https://python-docs-es.readthedocs.io/es/3.11/) on RtD. The source files aren't in the translation repo, so the English version is added as a submodule. Translations need to be added as a new project, then connected in the admin interface.
+  * For multiversion, the version switcher is injected on the bottom right. Each version is a branch or a tag, once enabled they get built.
+  * That said, RtD is rebuilding the flyout (switcher) to make it easier to customize. Working together with Pradyun (Furo, Lutra)
+
+* The “this is only a preview” header is missing.
+
+## Discussion
+
+* Building images with [mermaid.js](https://mermaid.js.org/) / [mermaid-cli](https://github.com/mermaid-js/mermaid-cli).
+  - @tysonclugg is trying to put together docs on concurrency which would benefit from good diagrams
+  - Some other tool to explore?
+
+* [Hugo] PEPs infra: PR opened to improve text readability / page performance
+https://github.com/python/peps/pull/3132
+
+* [Ned] Best entry point to working on docs, for people newly interested in helping?
+    - we need clear onboarding
+    - do we have style guide?
+    - Cristian: In the python-docs-es team we have a page inside the docs (docs-ception) that teach the whole process https://python-docs-es.readthedocs.io/es/3.11/CONTRIBUTING.html
+    - how do we write such docs?
+        - HackMD or Google Docs
+    - Python Docs Community web page, https://docs-community.readthedocs.io/en/latest/
+        - source: github.com/python/docs-community . e.g. front page source: [docs-community/docs/index.rst](https://github.com/python/docs-community/blob/main/docs/index.rst)
+        - start having better landing page for Docs there
+    - The landing page should point to where the conversations happen, e.g. the Discourse server. For a group of translators, a Telegram group is the conversation place. A conversation place helps people support each other.
+- Docs community Contributing Guide: https://docs-community.readthedocs.io/en/latest/community/contributing.html
+
+- Translation group
+    - how to coordinate with the wider translations group, share tips, etc
+    - Manuel: started 3 years ago: have a place to connect with other translators and resources
+    - Julien introduced the tools for translation.
+    - Create channel for translations
+    - Cristian: It also could be a good idea to have in the main Documentation page (that we will have soon) something like "Wanna help with a translation of these docs? check out this initiatives <list of languages here>"
+    - Ezio: I already brought it up a couple of times, but it would be good to unify the translation infrastructure and tools (see e.g. https://github.com/python/python-docs-zh-tw/issues/399)
```

Comment:
```suggestion
    - Ezio: I already brought it up a couple of times, but it would be good to unify the translation infrastructure and tools (see e.g. [python/python-docs-zh-tw#399](https://github.com/python/python-docs-zh-tw/issues/399))
```